svelte/src/libs/translations/index.ts

```diff
@@ -9,6 +9,7 @@ import ru from "./languages/ru.json";
 import uk from "./languages/uk.json";
 import zh from "./languages/zh.json";
 import ptbr from "./languages/ptbr.json";
+import pl from "./language/pl.json";
```

Comment:
@Gari7572 misssing an s

you can also test the ossapp locally by just executing 
```
xc setup

then

xc dev
```

assuming you pkgx cli https://pkgx.sh/